spec/system/court_dates/new_spec.rb

```diff
@@ -15,7 +15,7 @@
   before do
     travel_to now
     sign_in admin
-    visit casa_case_path(casa_case)
+    visit edit_casa_case_path(casa_case)
```

Comment:
Is there a reason why this had to be changed?

Reply:
This seems strange, what's up here?

Reply:
"add court date" button in casa_case_path isn't being rendered, so I assumed it was referring to the wrong page. I'll discover why it isn't rendering

Reply:
My bad, the css wasn't compiling correctly. Already corrected it.